provisioning/utils.go

```diff
@@ -113,7 +117,7 @@ func GetIronicIP(client kubernetes.Interface, targetNamespace string, config *me
 		ironicIP = inspectorIP
 	}
 
-	return
+	return ironicIP, inspectorIP, err
```

Comment:
nit: I don't think this change is needed

Reply:
When I do that, the unit tests throw error : `naked return in func "GetIronicIP" with 32 lines of code (nakedret)` , and explicit return also improves readability I think.